### DISCUSSION TITLE
[v4] Is there a short way to do `theme(colors.white / 10%)` with `vars()`?

### DISCUSSION THREAD

**Discussion by kerryj89** [tailwindlabs/tailwindcss#15304] [Help]
Since it is now recommended to use `var(--*)` instead of `theme(...)` where possible, is there some Tailwind abstraction shortcut to do this still with vars: 
`theme(colors.white / 10%)`

Or will we need to use `color-mix`?  I am fine with using the native color-mix, it's just more verbose but I can wrap that in a mixin.  But before I do anything I figured I'd ask.

**Comment by adamwathan** [comment]
For a lot of situations you can, like `bg-(--my-car)/25`.

Also works for arbitrary properties like `[--some-var:var(--other-var)]/25` 👍🏻

**Reply by kerryj89** [reply]
Thanks - and inside css we will just have to rely on `color-mix` right? I can't seem to find the right combo if there are any https://play.tailwindcss.com/bnGlhHn7yM?file=css
